### DISCUSSION TITLE
Can dynamic zoom  in Spot Mode tools?

### DISCUSSION THREAD

**Discussion by LY1806620741** [pubpub-zz/ppInk#312] [Ideas]
I want students to focus on something and zoom in dynamically.

![image](https://github.com/user-attachments/assets/32496908-c027-472f-a291-fbf1fe828a6d)

**Comment by pubpub-zz** [comment]
have a look at the spot mode and dynamic zoom
https://github.com/pubpub-zz/ppInk?tab=readme-ov-file#zoom

If you don't need the static zoom remember you can use the "frozen enlargement"

**Comment by pubpub-zz** [comment]
@LY1806620741 
is it ok ? can we close it ?

**Comment by LY1806620741** [comment]
I think I need dynamic zoom in Spot Mode.
like this.
![image](https://github.com/user-attachments/assets/4a607993-4526-405a-944f-080e08df4f5a)


frozen enlargement and rect static zoom tools feel a bit ugly

**Comment by pubpub-zz** [comment]
such a circular zoom is too tough for me to propose. Feel free to propose a PR if you have some ideas. I convert this thread an Idea discussion
